packages/console/src/components/Pagination/index.tsx

```diff
@@ -37,8 +46,15 @@ const Pagination = ({ pageIndex, totalCount, pageSize, className, onChange }: Pr
   const min = (pageIndex - 1) * pageSize + 1;
   const max = Math.min(pageIndex * pageSize, cachedTotalCount);
 
+  const pageRangeConfig = conditional(
+    isTinyStyle && {
+      pageRangeDisplayed: -1,
+      marginPagesDisplayed: 0,
+    }
+  );
```

Comment:
nit: Personally I would just apply these props on the JSX element.
```ts
const isPicoMode = mode === 'pico';

<ReactPaginate
 ...
 pageRangeDisplayed={conditional(isPicoMode && -1)}
 marginPagesDisplayed={conditional(isPicoMode && 0)}
/>
```

Reply:
updated with ternary operation for the conditional will cause an error.
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/10806653/210936391-43f43853-e385-47f8-828b-3b66d72698ee.png">